### ISSUE TITLE
Provide better UI for paths

### ISSUE THREAD

**Issue by asl** [asl/BandageNG#85]
* Model around GAF format: list all paths, some paths queries (like in #4)
  * Maybe join with BLAST hits and their paths

@ekg I started to draft better UI for paths. Unfortunately, I cannot check how the things are implemented in gfaestus – while after few hours I was able to compile it, I'm unable to run it.

**Comment by ekg** [comment]
Sorry that it's hard to run. FWIW, We are rewriting based on waragraph to
simplify the GPU related dependencies. But if you have a chance please
share your errors with Christian (author).

The key things are being able to go from a node to path positions and from
path positions to nodes. That's most of what gfaestus and waragraph (1D,
newer) aspire to both implement.

So the desired functionality would be 1) an interface that allows selection
of a path name and entry of a path position or range, selecting the nodes
in that range and zooming to them, and 2) given a selected node show what
paths and step positions are on it.

Bonus would be 3) BED input of many ranges -> colors and or labels on the
graph using the same positional mechanisms.

**Comment by asl** [comment]
Thanks!

> Sorry that it's hard to run. FWIW, We are rewriting based on waragraph to simplify the GPU related dependencies. But if you have a chance please share your errors with Christian (author).

Sure! But basically relying on Vulkan (for a non-game app) seems to be a culprit here. In the end I received "invalid driver" or something like this w/o clear understanding how to debug / fix.

**Comment by ekg** [comment]
Yup, it's awkward as a dependency, but Vulkan is what allows gfaestus to provide zoomable views of whole chromosome pangenome graphs with base-level variation. The rewrite in the works rebuilds things on a more portable subset of Vulkan.

Anyway, for this conversation the key thing is the functionality, which is just as well described in the abstract. Does it make sense what I'm suggesting?

**Comment by asl** [comment]
@ekg Yes, all this make perfect sense! Thanks!

**Comment by asl** [comment]
@ekg What do to exactly mean as `step` in _given a selected node show what paths and step positions are on it._?

**Comment by asl** [comment]
Ok, I think you just mean the node positions. This is WIP currently:
<img width="1461" alt="Screenshot 2022-07-24 at 17 07 38" src="https://user-images.githubusercontent.com/67392/180653500-73a26dee-87e3-4da9-8bb8-5483880ea3a2.png">

**Comment by asl** [comment]
Path position search is also done:
<img width="1577" alt="Screenshot 2022-07-24 at 18 45 11" src="https://user-images.githubusercontent.com/67392/180657556-37040f5a-8908-4f81-87fe-39341f532361.png">

**Comment by asl** [comment]
I'm going to keep this open in case more requests / ideas will appear

**Comment by gf777** [comment]
I was wondering if in the future one could think of a graphical way of representing the paths, ie. not only select/recolor but something that can highlight the direction and the multiple passes if the same node is involved multiple times. I think it could help when looking at tangles
